### DISCUSSION TITLE
close colour picker with enter key and keep colour

### DISCUSSION THREAD

**Discussion by skinofthesoul** [mdbassit/Coloris#151] [Ideas]
Hey there,
first off, I love Coloris! I just discovered that it's easy to navigate via keyboard, which is always a plus for me. However, there does not seem to be any way to pick a colour by keyboard and then close the picker by keyboard as well and **keep** that colour. So that's a pity for people who can't use a mouse so well. Maybe a good idea to add support for that? The enter key seems to me to be most intuitive here.

**Comment by mdbassit** [comment]
Hello, thank you for your kind words!

Actually there is a close button that is by default the little color preview disc next to the input field within the picker. You can also add a more obvious close button by setting the `closeButton` option to `true`. In both cases you can trigger the button with the enter and space keys

I will look into closing the color picker with the enter key, but that might interfere with the selection of a color swatch.

**Reply by skinofthesoul** [reply]
Oh, I see! If I press the Enter key on the actual colour field the colour is selected and the field is closed. That is absolutely perfect, thank you so much! No need to change your code 🙂
